src/Core/Repositories/EntityFramework/Queries/EmergencyAccessDetailsViewQuery.cs

```diff
@@ -0,0 +1,38 @@
+using System.Linq;
+using Bit.Core.Models.Data;
+
+namespace Bit.Core.Repositories.EntityFramework.Queries
+{
+    public class EmergencyAccessDetailsViewQuery : IQuery<EmergencyAccessDetails>
+    {
+        public IQueryable<EmergencyAccessDetails> Run(DatabaseContext dbContext)
+        {
+            var query = from ea in dbContext.EmergencyAccesses
+                join grantee in dbContext.Users
+                    on ea.GranteeId equals grantee.Id into grantee_g
+                from grantee in grantee_g.DefaultIfEmpty()
+                join grantor in dbContext.Users
+                    on ea.GrantorId equals grantor.Id into grantor_g
+                from grantor in grantor_g.DefaultIfEmpty()
+                select new {ea, grantee, grantor};
+            return query.Select(x => new EmergencyAccessDetails() {
+                Id = x.ea.Id,
+                GrantorId = x.ea.GrantorId,
+                GranteeId = x.ea.GranteeId,
+                Email = x.ea.Email,
+                KeyEncrypted = x.ea.KeyEncrypted,
+                Type = x.ea.Type,
+                Status = x.ea.Status,
+                WaitTimeDays = x.ea.WaitTimeDays,
+                RecoveryInitiatedDate = x.ea.RecoveryInitiatedDate,
+                LastNotificationDate = x.ea.LastNotificationDate,
+                CreationDate = x.ea.CreationDate,
+                RevisionDate = x.ea.RevisionDate,
+                GranteeName = x.grantee.Name,
+                GranteeEmail = x.grantee.Email,
+                GrantorName = x.grantor.Name,
+                GrantorEmail = x.grantor.Email
```

Comment:
Same here, trailing comma is needed. Not to be a pest about it, but consistency is best with such a large code push.